HomeWork_4/Main.java

```diff
@@ -0,0 +1,107 @@
+import java.util.Scanner;
+/**
+ * Main class which contained entry point.
+ * It read input from command line as three numbers.
+ * Solves the equation and prints results
+ * The program read only three number coming first. The following numbers are
+ * ignored.
+ *
+ * @author Nikita Zenchik
+ */
+public class Main {
+    /**
+     * Print message and receive three numbers from console.
+     */
+    // Constants
+    static final int CONSTANT1 = 4;
```

Comment:
It doesn't adequate names for these values. It's better to move it to class quadratic equation and all methods which relate to it